src/main/java/ch/njol/skript/effects/EffContinue.java

```diff
@@ -70,6 +70,7 @@ public String toString(@Nullable Event e, boolean debug) {
 		return "continue";
 	}
 
+	@SuppressWarnings("null")
 	@Override
```

Comment:
why did you add this

Reply:
Because there is a warning without it. Use the proper IDE to see.

Reply:
there's warnings all over skript. The class was already fine to ECJ.

Reply:
No it wasn't

Reply:
@TheLimeGlass I understand that you wanted good and making a separate PR would take too much effort/be pointless for only this change and I kinda feel you and agree (kinda because I wouldn't change this at all, this should be fixed along with the other 400 warnings IMO lol), but at least make a separate commit next time please. ~~It's easy and if it isn't then switch to IntelliJ because it is there~~

Reply:
That's.. not exactly.. what I meant